domains/scythemail.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "ScytheMediaTheDev",
+        "email": "tswartz22@aol.com"
+    },
+    "record": {
+        "MX": ["scythemail.is-a.dev"]
```

Comment:
You can't use the domain itself as MX record

Reply:
@Stef-00012 what do i do then, i dont know what to put as the record, new with domains